saleor/graphql/order/mutations/orders.py

```diff
@@ -483,3 +483,39 @@ def perform_mutation(cls, _root, info, amount, **data):
         if transaction.kind == TransactionKind.REFUND:
             order_refunded(order, info.context.user, amount, payment)
         return OrderRefund(order=order)
+
+
+class OrderConfirm(ModelMutation):
+    order = graphene.Field(Order, description="Order which has been confirmed.")
+
+    class Arguments:
+        id = graphene.ID(description="ID of an order to confirm.", required=True)
+
+    class Meta:
+        description = "Confirms an unconfirmed order by changing status to unfulfilled."
+        model = models.Order
+        permissions = (OrderPermissions.MANAGE_ORDERS,)
+        error_type_class = OrderError
+        error_type_field = "order_errors"
+
+    @classmethod
+    def get_instance(cls, info, **data):
+        instance = super().get_instance(info, **data)
+        if instance.status != OrderStatus.UNCONFIRMED:
+            raise ValidationError(
+                {
+                    "id": ValidationError(
+                        "Provided order id belongs to an order with status "
+                        "different than unconfirmed.",
+                        code=OrderErrorCode.INVALID,
+                    )
+                }
+            )
+        return instance
+
+    @classmethod
+    def perform_mutation(cls, root, info, **data):
+        order = cls.get_instance(info, **data)
+        order.status = OrderStatus.UNCONFIRMED
+        order.save(update_fields=["status"])
+        return OrderConfirm(order=order)
```

Comment:
1. Should we save any event in this mutation for example `OrderConfirmed`?
2. Should we trigger any webhook here?

Reply:
Yep, we should trigger a webhook, this is already done in separate PR.
Adding an event is a good idea, will do that in a moment.